var/spack/repos/builtin/packages/py-espressopp/package.py

```diff
@@ -27,7 +27,7 @@ class PyEspressopp(CMakePackage):
 
     depends_on("cmake@2.8:", type='build')
     depends_on("mpi")
-    depends_on("boost+serialization+filesystem+system+python+mpi", when='@1.9.4:')
+    depends_on("boost+serialization+filesystem+system+python+mpi cxxstd=11", when='@1.9.4:')
```

Comment:
What if we change the default in `boost`?

Reply:
Not sure, what that would break!

`py-esspressopp` is C++11 anyhow, so it doesn’t matter much.

Reply:
Not sure, what that would break!

`py-esspressopp` is C++11 anyhow, so it doesn’t matter much.